### ISSUE TITLE
avahi inside compose specs

### ISSUE THREAD

**Issue by rbroggi** [getumbrel/umbrel#1657]
Hello all, 

First of all, I wanted to thank you for this amazing product/project. In the spirit of eventually helping/contributing with it, I was wondering if you would think interesting/practical to try and support `avahi` from within the docker compose environment.  

I see there is already a nice image [here](https://github.com/flungo-docker/avahi) which we could refer to create our own (for safety purposes). 

I think that a natural added value of doing so is that we go one step further in making it easier for other linux distributions to run `umbrel`. 

WDYT? 

Best regards

**Comment by IMPranshu** [comment]
After inspecting the docker image, I don't think this has a web page frontend for GUI. I think it's a CLI.

**Comment by rbroggi** [comment]
Yes it's a mDNS daemon which is installed along umbrel (see the `install` script). Migrating from a specific package manager (`apt`) in favor of a bare docker container is more inclusive towards other linux distributions (and pave the way for an eventual other OS support)
